source/get-started/bootable-usb.rst

```diff
@@ -1,30 +1,19 @@
 .. _bootable-usb:
 
-Create a bootable USB drive
+Create a bootable USB drive using BalenaEtcher
 ###########################
 
-Follow the instructions applicable to your system to create a bootable
-|CL-ATTR| USB drive:
-
-* :ref:`bootable-usb-linux`
-* :ref:`bootable-usb-mac`
-* :ref:`bootable-usb-windows`
-
 Prerequisites
 *************
 
 * Download the |CL| Desktop or Server image from the `Downloads`_ page
 * Recommended minimum **4GB** USB drive or larger
+* Download and install `BalenaEtcher`_
```

Comment:
Probably need more detail here, maybe even it's own section. At the very least it should mention that it's a tool for making bootable USBs and Etcher is available for all the major OSs. 

Maybe even mention that Etcher is one tool shown for simplicity but others like Rufus or dd can be used too.